internal/state/helpers.go

```diff
@@ -20,51 +18,23 @@ const (
 	progressBarTpl    = `{{ etime . }} {{ bar . "" "=" }} {{ percent . }}`
 )
 
-func (c *State) Wrap(f func(*State, *cobra.Command, []string) error) func(*cobra.Command, []string) error {
+func Wrap(s State, f func(State, *cobra.Command, []string) error) func(*cobra.Command, []string) error {
```

Comment:
Because it would need to be reimplemented on each implementation of `state.State` even though it is a universal util function. The change was necessary because you can't define functions on interfaces.